docs/installation/presto-admin-installation.rst

```diff
@@ -6,8 +6,9 @@ Presto Admin Installation
 *Prerequisites:* `Python 2.6 or Python 2.7 <https://www.python.org/downloads>`_
 
```

Comment:
Meant to mention this, but forgot! There's a mixture of form in titles in the PR (Presto Admin Installation, and later on Configuring Presto, and some others). Suggest using -ing or nouns for consistency.